stonesoup/metricgenerator/base.py

```diff
@@ -36,6 +36,27 @@ class MetricManager(Base):
     """
 
 
+class MetricTableGenerator(Base):
```

Comment:
This should inherit from MetricGenerator, and then _generate_table_ could renamed _compute_metric_ method.